siege/front-proxy.yaml

```diff
@@ -0,0 +1,69 @@
+static_resources:
+  listeners:
+  # define an origin server on :10000 that always returns "lorem ipsum..."
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          codec_type: auto
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: service
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/"
+                direct_response:
+                  status: 200
+                  body:
+                    filename: "./lorem_ipsum.txt"
+          http_filters:
+          - name: envoy.router
+            config: {}
```

Comment:
Do you want https://github.com/envoyproxy/envoy-perf/blob/master/cloudperf/simple-loopback.json#L38 for these routers configs? Even when comparing relative performance, it's best not to skew the absolute numbers with things that are known not to be set when running high-performance benchmarks.

Reply:
Same also for https://github.com/envoyproxy/envoy-perf/blob/master/cloudperf/simple-loopback.json#L42

Reply:
Disable the request UUID generation and dynamic stats is considered best practice when benchmarking Envoy's data plane. Of course, one day maybe we want to benchmarking them as well, but I would make that a TODO for when we have templated configuration generation, which is definitely where we want to take this.